sherpa/optmethods/optfcts.py

```diff
@@ -583,6 +735,204 @@ def random_start(xmin, xmax):
 def neldermead(fcn, x0, xmin, xmax, ftol=EPSILON, maxfev=None,
                initsimplex=0, finalsimplex=9, step=None, iquad=1,
                verbose=0):
+    r"""Nelder-Mead Simplex optimization method.
+
+    The Nelder-Mead Simplex algorithm, devised by J.A. Nelder and
+    R. Mead [1]_, is a direct search method of optimization for
+    finding local minimum of an objective function of several
+    variables. The implementation of Nelder-Mead Simplex algorithm is
+    a variation of the algorithm outlined in [2]_ and [3]_. As noted,
+    terminating the simplex is not a simple task:
+
+    "For any non-derivative method, the issue of termination is
```

Comment:
Is this is directed quote from [2] or from [3]?

Reply:
I have no idea. This stuff is just copied over from ___init__.py

Reply:
Just to note for anyone reading this in the future that I'm not planning to address this issue in this PR.